### ISSUE TITLE
Persistent Layout behaviours

### ISSUE THREAD

**Issue by ammosov** [BladeMight/Mahou#358]
## Problem/Request description

Re https://github.com/BladeMight/Mahou/wiki/Functions-list

1. Можно ли добавить сюда объяснение того, что такое Persistent Layout ?
2. По умолчанию раскладка переключается для всей системы и при переходе из программы в программу клавиатура меняется. А существует ли включение поведения "запоминать и созранять настройку для каждой программы"?

**Comment by BladeMight** [comment]
1. Делает раскладку "запоминаемой", т.е. не зависит от переключения программ, Mahou запомнит какая была последняя и если в "программе на которую переключились" раскладка отличается - тогда поставит ту что запомнил.
2. Нет, это как было в Windows 7.

**Comment by ammosov** [comment]
1. Не понял до конца. Если слева включена галка, что происходит?
2. Какие настройки могут глобально эмулировать такое поведение?

**Comment by BladeMight** [comment]
1. Там есть описание в самом Mahou если навести курсор мыши на галочку.
2. [это](https://tunecomp.net/wp-content/uploads/2016/09/let-me-set-a-different-input-method-for-each-app-window.png)

**Comment by ammosov** [comment]
1. Вот я потому и написал, что там очень непонятно написано. 
2. В win10 последних версий эта опция переместилось в настройки клавиатуры, вроде бы
![image](https://user-images.githubusercontent.com/4894284/143860361-c4f4e48e-fc12-4535-9680-6f2a50716c8f.png)

**Comment by BladeMight** [comment]
1. Она работает как когда без "Let me use a different input...", в основном полезна для Win7.
2. Это так как на Win7, нет смысла включать "Постоянную раскладку", просто включи или выключи эту функцию.

**Comment by ammosov** [comment]
2. Не понял смысла написанного.

**Comment by BladeMight** [comment]
2. Ничего в Mahou смысла включать - нет, а в системе вкл./выкл. `Let me use a different input...` как тебе надо.

**Comment by ammosov** [comment]
Так. То есть включение в системе опции different input method приводит к override настроек Mahou, и после этого в каждом приложении сохранится своя раскладка? ОК. Теперь понятно. Я б это добавил в вики, или в раздел "как сделать".

**Comment by BladeMight** [comment]
> То есть включение в системе опции different input method приводит к override настроек Mahou, и после этого в каждом приложении сохранится своя раскладка?

Да, только наоборот:

Включение в Mahou опцию постоянная раскладка приводит к override настроек системной опции "different input method", как если бы она была отключена.
